.github/workflows/ci.yml

```diff
@@ -33,6 +33,30 @@ jobs:
         with:
           working-directory: src/github.com/containerd/nri
 
+  test-build:
+    name: Check buildability
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: src/github.com/containerd/nri
```

Comment:
I think we can delete gopath setting in the follow-up pull requests because we already use go.mod.